edge/pkg/edged/volume_host.go

```diff
@@ -135,9 +135,17 @@ func (evh *edgedVolumeHost) GetConfigMapFunc() func(namespace, name string) (*ap
 		return evh.edge.metaClient.ConfigMaps(namespace).Get(name)
 	}
 }
-func (evh *edgedVolumeHost) GetExec(pluginName string) utilexec.Interface  { return nil }
-func (evh *edgedVolumeHost) GetHostIP() (net.IP, error)                    { return nil, nil }
-func (evh *edgedVolumeHost) GetNodeAllocatable() (api.ResourceList, error) { return nil, nil }
+func (evh *edgedVolumeHost) GetExec(pluginName string) utilexec.Interface { return nil }
+func (evh *edgedVolumeHost) GetHostIP() (net.IP, error)                   { return nil, nil }
+
+func (evh *edgedVolumeHost) GetNodeAllocatable() (api.ResourceList, error) {
+	node, err := evh.edge.GetNode()
```

Comment:
Node returned here is an empty object

Reply:
Oh, i miss one change. It searches from metaClient indeed,